### PR TITLE
Legibility changes

### PR DESCRIPTION
**Pull request by Georacer**
This PR is two small legibility improvements in TECS.cpp

It also adds small typo fixes I did while reading TECS.cpp.

### CODE REVIEW

File: libraries/AP_TECS/AP_TECS.cpp [hunk 1]
```diff
@@ -749,7 +749,7 @@ void AP_TECS::_update_throttle_with_airspeed(void)
         // drag increase during turns.
         const float cosPhi = sqrtf((rotMat.a.y*rotMat.a.y) + (rotMat.b.y*rotMat.b.y));
         STEdot_dem = STEdot_dem + _rollComp * (1.0f/constrain_float(cosPhi * cosPhi, 0.1f, 1.0f) - 1.0f);
-        const float ff_throttle = nomThr + STEdot_dem / (_STEdot_max - _STEdot_min) * (_THRmaxf - _THRminf);
```
Comment (magicrub) [inline libraries/AP_TECS/AP_TECS.cpp:752]: this is not equivalent. The const var is doing / and this one is doing *.

**Review by magicrub — CHANGES_REQUESTED** [review]
The K_thr2STE maths is not equivalent, just remove that bit now. The other changes look OK

File: libraries/AP_TECS/AP_TECS.cpp (hunk 1 above)
Comment (Georacer) [reply libraries/AP_TECS/AP_TECS.cpp:752]: I did the math on my mind, with pencil and paper and here too:
```c
nomThr + STEdot_dem / (_STEdot_max - _STEdot_min) * (_THRmaxf - _THRminf) =
nomThr + STEdot_dem * (_THRmaxf - _THRminf) / (_STEdot_max - _STEdot_min) =
nomThr + STEdot_dem * ( (_THRmaxf - _THRminf) / (_STEdot_max - _STEdot_min) ) =
nomThr + STEdot_dem / ( (_STEdot_max - _STEdot_min) / (_THRmaxf - _THRminf) ) =
nomThr + STEdot_dem / K_thr2STE
```
Perhaps you can give it another look?

Comment (Georacer) [reply libraries/AP_TECS/AP_TECS.cpp:752]: Not sure why it fails the CI, though.

File: libraries/SITL/SITL.cpp [hunk 2]
```diff
@@ -208,7 +208,7 @@ const AP_Param::GroupInfo SIM::var_info2[] = {
     AP_GROUPINFO("TEMP_BFACTOR", 4, SIM,  temp_baro_factor, 0),
 
     AP_GROUPINFO("WIND_DIR_Z",  10, SIM,  wind_dir_z,     0),
-    // @Param: WIND_T_
+    // @Param: WIND_T
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:211]: Line needs to be removed in `vehicle_test_suite.py` - search for `WIND_T`

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: libraries/AP_TECS/AP_TECS.cpp [hunk 3]
```diff
@@ -566,7 +566,7 @@ void AP_TECS::_update_height_demand(void)
             _hgt_dem += _hgt_dem_tconst * _hgt_rate_dem;
         } else {
             // Don't allow height demand to get too far ahead of the vehicles current height
-                // if vehicle is unable to follow the demanded climb or descent
+                // if vehicle is unable to follow the demanded climb or descent
```
Comment (peterbarker) [inline libraries/AP_TECS/AP_TECS.cpp:569]: Bad whitespace change

File: libraries/AP_TECS/AP_TECS.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_TECS/AP_TECS.cpp:752]: Ping @magicrub

**Comment by peterbarker** [comment]
I tested this PR with this patch on top:
```
diff --git a/libraries/AP_TECS/AP_TECS.cpp b/libraries/AP_TECS/AP_TECS.cpp
index 9307502caaf..86a07566b7d 100644
--- a/libraries/AP_TECS/AP_TECS.cpp
+++ b/libraries/AP_TECS/AP_TECS.cpp
@@ -740,6 +740,10 @@ void AP_TECS::_update_throttle_with_airspeed(void)
         // Calculate gain scaler from specific energy error to throttle
         const float K_thr2STE = (_STEdot_max - _STEdot_min) / (_THRmaxf - _THRminf); // This is the derivative of STEdot wrt throttle measured across the max allowed throttle range.
         const float K_STE2Thr = 1 / (timeConstant() * K_thr2STE);
+        const float old_K_STE2Thr = 1 / (timeConstant() * (_STEdot_max - _STEdot_min) / (_THRmaxf - _THRminf));
+        if (!is_zero(K_STE2Thr-old_K_STE2Thr)) {
+            abort();
+        }
 
         // Calculate feed-forward throttle
         const float nomThr = aparm.throttle_cruise * 0.01f;
@@ -750,6 +754,10 @@ void AP_TECS::_update_throttle_with_airspeed(void)
         const float cosPhi = sqrtf((rotMat.a.y*rotMat.a.y) + (rotMat.b.y*rotMat.b.y));
         STEdot_dem = STEdot_dem + _rollComp * (1.0f/constrain_float(cosPhi * cosPhi, 0.1f, 1.0f) - 1.0f);
         const float ff_throttle = nomThr + STEdot_dem / K_thr2STE;
+        const float old_ff_throttle = nomThr + STEdot_dem / (_STEdot_max - _STEdot_min) * (_THRmaxf - _THRminf);
+        if (!is_zero(ff_throttle-old_ff_throttle)) {
+            abort();
+        }
 
         // Calculate PD + FF throttle
         float throttle_damp = _thrDamp;
```

... and there were not aborts.

Merging as agreed on DevCall last night.

**Pull request merged**